api/src/test/java/org/openmrs/PersonTest.java

```diff
@@ -824,6 +824,15 @@ private void checkGetPersonNameResultForVoidedPerson(PersonName expectedPersonAd
 		
 		assertEquals(expectedPersonAddress, actualPersonName);
 	}
+
+	@Test
+	public void shouldSetDeadToTrueIfSetDeathdate() {
+
+				Person p = new Person();
+				Date deathDate = new Date();
+				p.setDeathDate(deathDate);
+				assert (p.getDead());
```

Comment:
Yes I did it intentionally. This is because I noticed that the codacy bot raised it as an issue in my previous pull requests. Even though I didnt see any thing wrong with it at the time, I decided to use assert instead since functionality is almost the same and I thought it would enable me pass the codacy bot review